client/components/auth-form.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
```

Comment:
The Bootstrap CSS import only needs to be in the `App` component, from there it will work across all of your components. Also when importing something from `node_modules` we never directly path to it, you just use the name of the package. So move this import to the App component and update it to be:

```JAVASCRIPT
import 'bootstrap/dist/css/bootstrap.min.css';
```